### DISCUSSION TITLE
File Not Found

### DISCUSSION THREAD

**Discussion by Alina1111w** [vvip-68/LedPanelWiFi#83] [Вопросы и ответы]
Доброго времени суток, проблема следующая , залили прошику, потом файлы ( веб интерфейс ) но он так и не запускается 
Ниже часть вывода из монитора порта , но когда открываешь по адрессу 192...4.1 то пишет File Not Found
Спасибо за ранее!)
  I: Лень - двигатель прогресса
  J: Чем бы дитя не тешилось...
  K: С Рождеством!{S07.01.****}
  L: С днем Победы!{S09.05.**** 7:00#09.05.**** 21:30}
  M: Сегодня {D:dddd dd MMMM}, на улице {WS}, {WT}
  N: С праздником, соседи!
  O: Не скучайте!
  P: Счастья всем! И пусть никто не уйдет обиженным.
  Q: В отсутствии солнца сияйте сами!
  R: Раз! {C#000001}Два!! {C#FFFFFF}Три!!! {C#018C0A}Ёлочка,{C#FFFFFF} гори!!!{S01.01.****#19.01.****}{S01.12.****#31.12.****}
  S: Время принимать решения!
  T: Время делать выводы!
  U: Лето, ах лето!{S15.05.****#01.09.****}
  V: Нет предела совершенству!
  W: До {C#0080B0}Нового года{C#FFFFFF} осталось... понять как жить дальше...{S01.12.****#31.12.**** 23:59:59}
  X: Курочка по зёрнышку, копеечка к копеечке!
  Y: Подъем через {P7:30#Z#60#60#12345}!
  Z: -Доброе утро!

Папка '/web' обнаружена.
Cписок файлов web-интерфейса:
   [web]
      [assets]

**Comment by vvip-68** [comment]
Вы не залили файлы веб-интерфейса - список пуст.
Во всей видимости вы используете Arduino IDE версии 2.x.x и его плагин для загрузки файловой системы - а они глючные до невозможности. Многие пользователи жалуются на них. Пробуйте еще раз 10 - может быть повезет. Тогда вот этот список папок web и assets будет содержать файлы.

Либо вы выбрали разбивку файловой системы с недостаточным местом под файлы и плагин просто не может их загрузить в файловую систему микроконтроллера.

У некоторых пользователей (не знаю как и чем они загружали) плагины вообще выводили ошибки при сжатии файлов в пакет перед загрузкой. Проверьте нет ли ошибок у вас. Плагины берете где-то на стороне или из папки tools проекта?

Если загрузить в 2.x.x так и не удастся - поставьте старую проверенную IDE 1.8.19 и плагины к ней для ее версии - попробуйте загрузить ими.
